webflux5-sleuth/pom.xml

```diff
@@ -21,6 +21,7 @@
 
     <spring-boot.version>2.3.7.RELEASE</spring-boot.version>
     <sleuth.version>2.2.6.RELEASE</sleuth.version>
+    <zipkin-reporter.version>${old-zipkin-reporter.version}</zipkin-reporter.version>
```

Comment:
sleuth uses an old version of zipkin-reporter and as it is unmaintained, probably won't update. Brave 6 removes version management from reporter for reasons like this.